### ISSUE TITLE
BioJS integration

### ISSUE THREAD

**Issue by piotr-gawron** [ebi-uniprot/ProtVista#42]
I learnt about this component when it was in the BioJS registry. I noticed that right now there is something wrong with the component in BioJS registry. Neither of the two components that link to this project work:
* http://biojs.io/d/biojs-vis-proteinfeaturesviewer
* http://biojs.io/d/ProtVista

Also technical links to technical documentation from original news post on http://www.ebi.ac.uk/about/news/service-news/new-view-proteins doesn't work.
(the original link was pointing to: http://ebi-uniprot.github.io/biojs-vis-proteinFeaturesViewer/developerGuide.html).

**Comment by ljgarcia** [comment]
Hi,

Thanks for pointing this out. We have identified the problem regarding biojs.io and we are working with them in order to solve it. As for the documentation link, sadly it looks like there is not too much we can do, pages are not re-linked after renaming a repo. We are still investigating though.

**Comment by ljgarcia** [comment]
We have contacted BioJS team. There is a problem regarding component with capital letters. It has been solved (http://dev.biojs.io/d/ProtVista) but it is not in production yet. Not much we can do about it. There is still a problem with the styles, we are investigating that.

As for the documentation, sadly GitHub does not redirect pages whenever a repository is renamed :-( We are still looking at some options.
